pkgs/applications/graphics/draftsight/default.nix

```diff
@@ -54,12 +85,12 @@ stdenv.mkDerivation {
   src = requireFile {
     name = "draftSight.deb";
     url = "https://www.3ds.com/?eID=3ds_brand_download&uid=41&pidDown=13426&L=0";
```

Comment:
ArchLinux is using the following URL: http://dl-ak.solidworks.com/nonsecure/$pkgname/$pkgver/draftSight.rpm, see: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=draftsight#n32 might be a better choice as it includes the version.

Reply:
It seems "safer" to respect the screen, in general anyway.  Maybe check with upstream first? It's kind of annoying & user unfriendly but if that's the way upstream intends it, that should be respected.